Formula/kollector.rb

```diff
@@ -0,0 +1,32 @@
+
+class Kollector < Formula
+  desc "Targeted de novo Gene Assembly"
+  homepage "https://github.com/bcgsc/kollector"
+  url "https://github.com/bcgsc/kollector/archive/V1.0.tar.gz"
+  version "1.0"
```

Comment:
Explicit version shouldn't be necessary, the URL is simple enough for Homebrew to pick it up.